Add support for skipping headings

pasky opened the pull request:
Hi! For my purposes, the text extracted from Wikipedia should not include headings but only real sentences. I added a -H switch that will make the script skip the headings.

pasky committed:
Add support for -H (--no-headings) commandline option

pasky committed:
WikiExtractor.py: +x mode

bwbaugh commented:
Looks useful. However, since this repo is simply mirroring the [official site](http://medialab.di.unipi.it/wiki/Wikipedia_Extractor), if you'd like this to get merged in then I would suggest contacting one of the authors.

I should probably update this repository's `README` to indicate the same for others.

pasky committed:
Add support for -x (--output-xml) commandline option

Without this post-processing, interpreting the output as XML is
very problematic; e.g. <br>, <poem>...</poem> etc. seeps through,
as well as lonely & characters.

pasky committed:
Expand {{convert}} templates extremely simplistically

They cover a large portion of factual numeric data in enwiki, so it
merits special handling.

Arkanosis commented:
Hello,
FYI, Giuseppe Attardi has set up an official repository at [attardi/wikiextractor](/attardi/wikiextractor). You might want to re-submit your pull request there.
Best regards

bwbaugh commented:
This is a mirror repo (mainly for history before the author moved to github).

If the issue is still outstanding, please consider making an issue on the official repo: https://github.com/attardi/wikiextractor

---

For what it's worth, it looks like the official repo has a newer version of the script than the [wiki website](http://medialab.di.unipi.it/wiki/Wikipedia_Extractor) that this repo is mirroring.
